packages/components/src/Dropzone/utils.ts

```diff
@@ -0,0 +1,101 @@
+import { ErrorCode, FileError } from "react-dropzone"
+import { DropzoneErrorsProps } from "./sub-components/DropzoneError/DropzoneError"
+
+export const trunicateFilename = (
+  filename: string,
+  maxCharacterLimit: number = 50
+): string => {
+  const fileNameArray = filename.split(".")
+  const fileExtension = fileNameArray[fileNameArray.length - 1]
+  const name = filename.slice(0, -fileExtension.length)
+  const shortenedFileName = name
+    .slice(0, maxCharacterLimit)
+    /**
+     * using brackets here to prevent user confusion with the file extension '.'
+     * this: filename[...].xlsx
+     * instead of: filename....xlxs
+     */
+    .padEnd(maxCharacterLimit + 5, "[...]")
+
+  return `${shortenedFileName}.${fileExtension}`
+}
+
+export const prettifyErrorCode = (errorCode: ErrorCode | string): string => {
+  let title: string = ""
+  /** Capitalise first letter */
+  title = errorCode.charAt(0).toUpperCase() + errorCode.slice(1)
+  /** Replace all hyphens with spaces */
+  title = title.replace(/-/g, " ")
+  return title
+}
+
+export const printAcceptedFileTypes = (
```

Comment:
This function would need to work on the `accept` object rather than an array of strings as we'd want to make sure this scales to more than just one file type.